src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title,
+  date,
+  text,
+}) {
+  return (
```

Comment:
let's use arrow syntax here
```suggestion
export const Article = ({
  title,
  date,
  text,
}) => (
```